### DISCUSSION TITLE
Multicolor printing with a single extruder?

### DISCUSSION THREAD

**Discussion by Lucas5004** [OrcaSlicer/OrcaSlicer#1137] [Q&A]
I am currently trying to print something on my printer that will be printed with two different filaments. I have specified two filaments in the slicer and also colored my object. Unfortunately, no filament is taken into account when printing.@SoftFever

**Comment by hliebscher** [comment]
witch printer / OS ? we need more information / i use this daily

**Reply by Lucas5004** [reply]
I have a lotmaxx sc-10 and an anycubic kobra plus windows 10
